zeppelin-web/pom.xml

```diff
@@ -120,34 +152,77 @@
           </execution>
 
           <execution>
-            <id>yarn install</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>install --no-lockfile</arguments>
-            </configuration>
+              <id>yarn add bower</id>
+              <goals>
+                  <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <!--ignore-scripts-->
+                  <arguments>add bower --no-lockfile --no-bin-links</arguments>
```

Comment:
Current master branch build package with

```
mvn package
```

and run test with
```
mvn test
```

and it does not install nodejs, yarn, bower again, unless user run `mvn clean`.

So, I'm not quite understand what you're trying to achieve here by running command from maven instead of npm script. Or what problem do you have when you run command from npm script instead of maven. Can you elaborate little more?